primitives/message-lane/src/source_chain.rs

```diff
@@ -16,14 +16,26 @@
 
 //! Primitives of message lane module, that are used on the source chain.
 
-use crate::{InboundLaneData, LaneId};
+use crate::{InboundLaneData, LaneId, MessageNonce};
 
-use frame_support::Parameter;
-use sp_std::fmt::Debug;
+use frame_support::{Parameter, RuntimeDebug};
+use sp_std::{collections::btree_map::BTreeMap, fmt::Debug};
 
 /// The sender of the message on the source chain.
 pub type Sender<AccountId> = frame_system::RawOrigin<AccountId>;
 
+/// Relayers rewards, grouped by relayer account id.
+pub type RelayersRewards<AccountId, Balance> = BTreeMap<AccountId, RelayerRewards<Balance>>;
```

Comment:
It's kinda confusing that the `RelayersRewards` typedef and `RelayerRewards` struct are name _almost_ the same. Maybe we can rename `RelayerRewards` to just `Reward`?

Reply:
I don't like `Reward` :) Imo using `item` and `items` for container of `item`-s is fine, even though they're almost the same. I could change e.g. `RelayersRewards` to `RelayersRewardsMap` if you feel it is better.

Reply:
I don't think adding the type to the name is a good idea either. What if we decide in the future that this should be a `Vec<Reward>` instead? Unfortunately I can't think of any good suggestions other than this:

```
pub type RelayerRewards<AccountId, Balance> = BTreeMap<AccountId, Reward<Balance>>;
```